client/components/theme-tier/theme-tier-badge/theme-tier-partner-badge.js

```diff
@@ -0,0 +1,39 @@
+import { PremiumBadge } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'calypso/state';
+import { isMarketplaceThemeSubscribed } from 'calypso/state/themes/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import useThemeTier from '../use-theme-tier';
+import ThemeTierBadgeTracker from './theme-tier-badge-tracker';
+
+export default function ThemeTierPartnerBadge( { themeId } ) {
+	const translate = useTranslate();
+	const siteId = useSelector( getSelectedSiteId );
+	const isPartnerThemePurchased = useSelector( ( state ) =>
+		siteId ? isMarketplaceThemeSubscribed( state, themeId, siteId ) : false
+	);
+	const { isThemeAllowedOnSite } = useThemeTier( siteId, themeId );
+
+	if ( isPartnerThemePurchased ) {
+		return <span>{ translate( 'Included in my plan' ) }</span>;
```

Comment:
Could this be clearer? It's technically included in your "subscription to this theme plan", but..

Not a blocker for merging, but is something we should address later when we have design input.